### ISSUE TITLE
Problem when query Host

### ISSUE THREAD

**Issue by Unterstrichmoepunterstrich** [Nosmoht/ansible-lookup-plugin-ipa#1]

**Comment by Unterstrichmoepunterstrich** [comment]
Hello,

I have an IPA with several host groups. e.g. servers or ipaservers.
So I changed the YAML code into ` ipa_group: '{{ lookup("ipaservers")}}'` and runned ansible.
Itts checking my current Hostvars and here I get this output:
`
TASK [get IPA group information for ipaservers]
fatal: [ipa.xxx]: FAILED! => {"msg": "lookup plugin (ipaservers) not found"}
`
Whats the problem?

**Comment by Nosmoht** [comment]
Did you add the path to your ansible.cfg?

**Comment by Unterstrichmoepunterstrich** [comment]
Yes I did, but when I do `ansible --version` the path isn't showed. Do you have any suggestions?
I found some Ansible 2.x issues. I'm using 2.5.

btw: does your role write uses it "on the fly" or is it nessecary for me to write it in a file and save it where my hosts file is stored?

thx
